zppy/mpas_analysis.py

```diff
@@ -31,8 +31,13 @@ def mpas_analysis(config: ConfigObj, script_dir: str, existing_bundles, job_ids_
     # job should run at once. To gracefully handle this, we make each
     # MPAS-Analysis task dependant on all previous ones. This may not
     # be 100% fool-proof, but should be a reasonable start
-    dependencies = []
+
+    # Dependencies carried over from previous task.
+    carried_over_dependencies: List[str] = []
+
     for c in tasks:
+
+        dependencies: List[str] = carried_over_dependencies
```

Comment:
`mpas_analysis` doesn't have any other dependencies besides previous runs of itself, so we could just move this definition out of the `for c in tasks` loop, but I think it's more comprehensible to explicitly note the dependency on previous runs is being carried over, via the name `carried_over_dependencies`